pkgs/development/tools/misc/edb/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, boost, capstone
+, double-conversion, graphviz, qtxmlpatterns }:
+
+stdenv.mkDerivation rec {
+  pname = "edb";
+  version = "2019-04-04";
+
+  src = fetchFromGitHub {
+    owner = "eteran";
+    repo = "edb-debugger";
+    rev = "77e5ebf497ff489827fe1b4b5861a77688aa3729";
+    fetchSubmodules = true;
+    sha256 = "1pfz65q5p1cpr2jz8mwpdhyckk7rx21inha4a65ykpim745n18c5";
+
+    # The build script checks for the presence of .git to determine whether
+    # submodules were fetched and will throw an error if it's not there.
+    leaveDotGit = true;
+  };
+
+  buildInputs = [
+    cmake pkgconfig boost.dev capstone double-conversion graphviz qtxmlpatterns
```

Comment:
nitpick: cmake and pkgconfig should be `nativeBuildInputs` instead of `buildInputs`.

Reply:
Done